similarity/src/main/java/edu/illinois/cs/cogcomp/config/SimConfigurator.java

```diff
@@ -12,21 +12,21 @@
 import edu.illinois.cs.cogcomp.core.utilities.configuration.ResourceManager;
 
 public class SimConfigurator extends Configurator {
-	public static final Property PHRASE_DICT = new Property("phrase_dict", "data/phrases");
-	public static final Property USE_NE_COMPARISON = new Property("useNES", Configurator.FALSE);
+	public static final Property PHRASE_DICT = new Property("phrase_dict", "phrases.txt");
+	public static final Property USE_NE_COMPARISON = new Property("useNER", Configurator.FALSE);
 	public static final Property USE_PHRASE_COMPARISON = new Property("usePhraseSim", Configurator.FALSE);
 	public static final Property USE_SIMPLE_SCORE = new Property("useSimpleScore", Configurator.FALSE);
 	public static final Property STOPWORD_FILE = new Property("stopwordFile", "llmStopwords.txt");
 	public static final Property WORD_METRIC = new Property("wordMetric", "wordnet");
 	public static final Property WORD_ENTAILMENT_THRESHOLD = new Property("wordEntailmentThreshold", "0.001");
 	public static final Property LLM_ENTAILMENT_THRESHOLD = new Property("llmThreshold", "0.5");
-	public static final Property WORD2VEC = new Property("word2vec", "");
-	public static final Property PARAGRAM = new Property("paragram", "data/paragram_vectors.txt");
-	public static final Property GLOVE = new Property("glove", "");
-	public static final Property PHRASE2VEC = new Property("phrase2vec", "");
-	public static final Property MEMORYBASEDESA = new Property("memorybasedESA", "");
```

Comment:
check paths are consistent with README, tests, any classes that run the sim code